glue/src/it/scala/com/gu/tableversions/glue/GlueMetastoreSpec.scala

```diff
@@ -0,0 +1,88 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.IO
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth._
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.glue.{AWSGlue, AWSGlueClient}
+import com.gu.tableversions.core.Partition.PartitionColumn
+import com.gu.tableversions.core._
+import com.gu.tableversions.metastore.MetastoreSpec
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cats.implicits._
+import com.amazonaws.services.glue.model._
+
+import scala.util.Random
+
+class GlueMetastoreSpec extends FlatSpec with Matchers with BeforeAndAfterAll with MetastoreSpec {
+
+  lazy val region = Regions.EU_WEST_1
+
+  lazy val credentials = new AWSCredentialsProviderChain(
+    new EnvironmentVariableCredentialsProvider(),
+    new SystemPropertiesCredentialsProvider(),
+    new ProfileCredentialsProvider("ophan"),
+    new ProfileCredentialsProvider(),
+    new InstanceProfileCredentialsProvider(false)
+  )
+  val glue: AWSGlue = AWSGlueClient.builder().withCredentials(credentials).withRegion(region).build()
+  val schema = "temp"
+  def tableLocation(tableName: String) = new URI(s"s3://ophan-temp-schema/test-data/$tableName")
```

Comment:
good point, we could put the AWS profile name there as well.

To be honest table locations don't really matter here and Glue doesn't seem to complain if you create tables with fake paths.
I was just trying to avoiding any unforeseen issues pointing the tables to a bucket that exists and we own but it could be anything really.